lib/llvmopencl/LLVMUtils.h

```diff
@@ -104,10 +121,9 @@ void CloneFunctionIntoAbs(llvm::Function *NewFunc,
                           llvm::ValueMapTypeRemapper *TypeMapper = nullptr,
                           llvm::ValueMaterializer *Materializer = nullptr) {
 
-
-#ifdef LLVM_OLDER_THAN_13_0
-  CloneFunctionInto(NewFunc, OldFunc, VMap, true,
-                    Returns, NameSuffix, CodeInfo, TypeMapper, Materializer);
+#if LLVM_MAJOR < 13
+  CloneFunctionInto(NewFunc, OldFunc, VMap, true, Returns, NameSuffix, CodeInfo,
```

Comment:
Which one is LLVM style capital or non-capital function/method names?

Reply:
functions should start with lower case, but it seems LLVM is not very good at following its own rules: https://llvm.org/doxygen/Cloning_8h.html - CloneFunctionInto is from LLVM header